### PR TITLE
Export an actually working es6 module

### LINKED ISSUE #3: cannot be loaded as a native browser esm
Hi,

```import xbytes from "/node_modules/xbytes/dist/index.js"```
just won't work because there are no exported symbols.

See for example [how flatpickr declares it in package.json](https://github.com/flatpickr/flatpickr/blob/d5c3029120708636b37d780810899a02af6d03b3/package.json#L53-L54).

### PR DESCRIPTION
**Pull request by kapouer**
Fix #3

### CODE REVIEW

**Comment by kapouer** [comment]
This assumes `npm run preversion` is run from a reasonably bash-compatible shell...

File: package.json [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
+  "exports": {
+    "require": "./dist/index.js",
+    "import": "./dist/index.mjs"
+  },
   "types": "typings/index.d.ts",
   "scripts": {
     "test": "jest",
     "build": "babel lib -d dist",
-    "preversion": "npm run build",
+    "module": "echo '\nexport default xbytes;' | cat dist/index.js - > dist/index.mjs",
```
Comment (miraclx) [inline package.json:14]: This should probably be part of the build task

Comment (miraclx) [reply package.json:14]: So it's a single `npm run build` for people who also want to use it as a module as opposed to following that with a second `npm run module`

Comment (kapouer) [reply package.json:14]: Well as you can see `npm run module` is called from `npm run preversion`.
Mind that i think `preversion` is not the right name here. It's supposed to be `prepare` in npm semantics.

Comment (kapouer) [reply package.json:14]: Documentation:
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#devdependencies

Comment (kapouer) [reply package.json:14]: Also note that ".mjs" extension is carefully chosen here. Using any other extension will let also nodejs fail to load it as es module (i tried .es.js for example and it fails).
See also https://github.com/nodejs/node/issues/34515 for further info.

Comment (kapouer) [reply package.json]: I found a command that is simpler to understand, and added it to "build" target.

**Comment by miraclx** [comment]
lgtm; Thanks for this.

**Comment by miraclx** [comment]
Does the readme need an update? https://github.com/miraclx/xbytes#usage

**Comment by kapouer** [comment]
I don't think so - i made it so it doesn't change anything w.r.t. usage.

**Comment by miraclx** [comment]
Great. All good then.

**Comment by kapouer** [comment]
FYI you can do a test with a test.mjs file (provided node_modules/xbytes is present)
```
import xbytes from "xbytes"
console.log(xbytes(100000));
```
then
> $ node test.js
> (node:710173) ExperimentalWarning: The ESM module loader is experimental.
> 100.00 KB

(that's with node 12).

**Pull request merged**